AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -76,6 +76,16 @@ final class CoreDataMeasurementStreamStorage: MeasurementStreamStorage {
         newStream.gotDeleted = false
 
         session.addToMeasurementStreams(newStream)
+        
+        let existingThreshold: SensorThreshold? = try context.existingObject(sensorName: stream.sensorName ?? "")
```

Comment:
I'm not sure I like those `?? ""` silent fallbacks here. I know those `nil` values are often a PITA, but maybe since this is already a `throwing` function we should guard against it and handle errors?

```Swift
guard let sensorName = stream.sensorName else {
    throw StorageError.malformedStream(context: "Stream does not have a sensor name!")
}
```

Or is it purposeful here? Than maybe to emphasize it:

```Swift
let sensorName = stream.sensorName ?? ""
```

Reply:
Hmm, dunno how it happened but we don't need it at all. It's not optional and when I'm using guard let I get an error that stream.sensorName isn't optional. 
I removed silent fallbacks, too :)

Reply:
I know why, it's of a different type `StreamOutput` while the stream below is of a type `MeasurementStream`.